### ISSUE TITLE
Feature suggestion: Multiple apps on a single machine with reverse proxy routing or Kubernetes

### ISSUE THREAD

**Issue by timbrandin** [arunoda/meteor-up-legacy#553]
I noticed that you're using docker containers in MUPX, why aren't MUPX instead setting up a Kubernetes cluster which we in turn can deploy custom images with our apps to?

Would be nice to have multiple small apps running on the same machine!

**Comment by arunoda** [comment]
Aka: mini Galaxy.
The reason we use docker is to avoid issues raised by different
environments.

Mup was not targeted as a tool to deploy multiple apps on a single server.
Although we can do it.

**Comment by timbrandin** [comment]
Yeah an open source Mini-Galaxy would be nice but I'd keep the current name
(mup) as it is already very well known and has much writing about it
already on the web.

It would be super nice feature anyhow, to just target a server to deploy to
and it puts it in that cloud. :)

2015-07-21 16:59 GMT+02:00 Arunoda Susiripala notifications@github.com:

> Aka: mini Galaxy.
> The reason we use docker is to avoid issues raised by different
> environments.
> 
> Mup was not targeted as a tool to deploy multiple apps on a single server.
> Although we can do it.
> On 2015 ජූලි 21, අඟහ at ප.ව. 7.17 Tim Brandin notifications@github.com
> wrote:
> 
> > I noticed that you're using docker containers in MUPX, why aren't MUPX
> > instead setting up a Kubernetes cluster which we in turn can deploy
> > custom
> > images with our apps to?
> > 
> > Would be nice to have multiple small apps running on the same machine!
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/arunoda/meteor-up/issues/553.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/arunoda/meteor-up/issues/553#issuecomment-123359816.

## 

Cheers,
// Tim Brandin @timbrandin https://twitter.com/#!/timbrandin
http://timbrandin.com
http://studiointeract.com

**Comment by gumyn** [comment]
+1 
Would it be bad for MDG business ?
